h/static/scripts/annotation-ui.js

```diff
@@ -1,8 +1,11 @@
 'use strict';
 
+var immutable = require('seamless-immutable');
+var redux = require('redux');
+
 function value(selection) {
```

Comment:
I was trying to keep the name consistent with the import name but I agree that `freeze()` would be a better name given how we're using it. The docs for 'seamless-immutable' use `Immutable` as the import name (as if it were a constructor) but it isn't really.